### ISSUE TITLE
DLS: Striped Table with a th on a row doesn't meet color contrast standards

### ISSUE THREAD

**Issue by lalov1** [utsa-asc/college-dls#782]
https://utsa-asc.github.io/college-dls/components/detail/table--striped-blue.html

![Image](https://github.com/utsa-asc/college-dls/assets/7785251/f7c28185-82bf-4bf5-bfb5-f36aef4d7652)

**Comment by garza-utsa** [comment]
thankfully,  I guess this is why Simpson Scarborough declared so many grey

**Comment by garza-utsa** [comment]
@Uri-E ,  should we use a lighter grey here? I imagine we might also need to check any <anchor styling as well

**Comment by garza-utsa** [comment]
going to move to $blue for th on blue and grey table designs, as our lightest off-white ($white-b) still doesn't provide enough contrast with $orange-a11y (only 4.24, min is 4.5)

**Comment by lalov1** [comment]
One thing I noticed is the Tabs and Accordions have the same background as the new grey, so they don't show the stripes: https://uc.utsa.edu/programs/mdst-ba/mdws.html
